cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabasePrivilegeTest.scala

```diff
@@ -98,7 +98,7 @@ DENY DROP CONSTRAINT ON DATABASE * TO my_role
 
 Deny privilege to drop constraints on all databases to a role.
 
-###assertion=update-two
+###assertion=update-zero
```

Comment:
This test seems a bit weird now, since it actually doesn't achieve what it says.

Reply:
It is the ref card so I think it is more that we want to show the command (and I don't know how to set up initial privileges so there is something to revoke here)